app/assets/stylesheets/dlme.scss

```diff
@@ -340,8 +351,12 @@ body {
     border: 0;
   }
 
-  .masthead .site-title {
-    text-align: center;
+  .masthead .site-title-container {
+    display: none;
```

Comment:
Can you add a comma here with the reason we're not displaying it?

Reply:
added